ui/common/ProgressBar.tsx

```diff
@@ -0,0 +1,21 @@
+'use client'
+
+import clsx from 'clsx'
+
+export default function ProgressBar({ progress }: { progress: number }) {
+  return (
+    <div className="relative h-[30px] w-full rounded-[5px]  bg-black/20">
+      <div
+        className={clsx(
+          'absolute top-0 left-0 h-full transform-gpu rounded-[5px] bg-[#FBEBC6] transition-transform',
+          {
+            '!bg-[#5DBC59]': progress === 100,
+          }
+        )}
+        style={{
+          width: `${progress <= 0.5 && progress > 0 ? 0.5 : progress}%`,
```

Comment:
Earlier in this PR I posted a link to a codesandbox that has a progress bar which animates using transforms which do not cause paints when `progress` changes, modifying the `width` does cause paints. If you can, please try to use transforms only to make full use of the GPU.

You can turn this on in your devtools to see whether your animation causes any paints:
![image](https://github.com/saving-satoshi/saving-satoshi/assets/10757768/a2d34460-0aa3-4ae4-ba76-7459503460ae)

And you can learn more about this subject [here](https://web.dev/animations-overview/) and [here](https://web.dev/animations-guide/).

Reply:
based on that sandbox I added `transform-gpu` and `transition-transform`. I tried out the Paint flashing devtool in Chrome and it still displays the progress bar with green highlights.

Reply:
That's probably due to rerendering of the component itself, lgtm for now, I can also have a look later